assets/js/main.js

```diff
@@ -49,23 +50,26 @@ import(/* webpackChunkName: "vue-apps" */ './vue-apps/index').then(vueApps => {
     vueApps.init();
 });
 
-const isDoNotTrack =
-    window.doNotTrack === '1' ||
-    window.navigator.doNotTrack === '1' ||
-    window.navigator.msDoNotTrack === '1';
+function shouldInitAnalytics() {
+    const isDoNotTrack =
+        window.doNotTrack === '1' ||
+        window.navigator.doNotTrack === '1' ||
+        window.navigator.msDoNotTrack === '1';
 
-/* Disable analytics outside of the real domain to avoid polluting data. */
-function shouldBlockAnalytics() {
-    const isProduction = window.AppConfig.environment === 'production';
-    const isProdDomain =
-        window.location.hostname === 'www.tnlcommunityfund.org.uk';
-    return !isProduction && !isProdDomain;
+    if (window.AppConfig.environment === 'production') {
+        /*
+         * In production, disable analytics outside
+         * of the real domain to avoid polluting data.
+         */
+        return (
+            window.location.hostname === 'www.tnlcommunityfund.org.uk' &&
+            isDoNotTrack === false
+        );
+    } else {
+        return isDoNotTrack === false;
+    }
 }
 
-if (!shouldBlockAnalytics() && !isDoNotTrack) {
-    import(/* webpackChunkName: "analytics" */ './analytics').then(
-        analytics => {
-            analytics.init();
-        }
-    );
+if (shouldInitAnalytics() === true) {
+    analytics.init();
```

Comment:
We were code-splitting this but it was a ~2kb module so really isn't needed. Inlining this to save the extra network request.